docs/reference/es_compatible_api.md

```diff
@@ -158,6 +196,31 @@ The payload is expected to alternate:
 - a `header` json object, containing the targetted index id.
 - a `search request body` as defined in the [`_search` endpoint section].
 
+
+### `_scroll` &nbsp; Scroll API
+
+```
+GET api/v1/_elastic/_search/scroll
+```
+
+#### Supported Request Body parameters
+
+| Variable      | Type       | Description                                                      | Default value |
+|---------------|------------|------------------------------------------------------------------|---------------|
+| `scroll_id`     | Scroll id (obtained from a search response)  | Required
+
+
+The `_scroll`, in combination with the `_search` API makes it possible successive pages of search results.
+First, the client needs to call the `search api` with a `scroll` query parameter, and then pass the `scroll_id` returned in the response payload to  `_scroll` endpoint.
```

Comment:
I couldn't find corresponding docs on the search API. Also `_scroll` should probably be replaced with `_search/scroll`.